hostscripts/clouddata/syncSLErepos

```diff
@@ -180,4 +180,18 @@ for arch in x86_64 aarch64 s390x ; do
     fi
 done
 
+# reduce disk usage
+fdupes -q -n -r /srv/nfs/repos/ |
```

Comment:
could add `/srv/nfs/install` because there is also duplication:
3aa21bf72846d40dd3e1f54f070b9177  repos/x86_64/SLES12-SP2-Pool/x86_64/zypper-1.13.10-12.7.x86_64.rpm
3aa21bf72846d40dd3e1f54f070b9177  install/suse-12.2/x86_64/install/suse/x86_64/zypper-1.13.10-12.7.x86_64.rpm